applications/GeoMechanicsApplication/custom_utilities/scheme_factory.hpp

```diff
@@ -0,0 +1,41 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#pragma once
+
+#include "solving_strategies/schemes/scheme.h"
+#include "custom_strategies/schemes/backward_euler_quasistatic_U_Pw_scheme.hpp"
+#include <memory>
+
+namespace Kratos
+{
+
+template<class TSparseSpace, class TDenseSpace>
+class SchemeFactory
+{
+public:
+    static std::shared_ptr<Scheme<TSparseSpace, TDenseSpace>> Create(const Parameters& rSolverSettings)
```

Comment:
Similar to my previous comment, I think that adding an alias improves readability here:
```suggestion
    using SchemeType = Scheme<TSparseSpace, TDenseSpace>;
    static std::shared_ptr<SchemeType> Create(const Parameters& rSolverSettings)
```

Reply:
Agreed, done